schema.sql

```diff
@@ -18,3 +18,6 @@ ALTER TABLE animals ADD COLUMN owners_id INT REFERENCES owners(id);
 CREATE TABLE vets (id serial PRIMARY KEY NOT NULL, name VARCHAR(50) NOT NULL, age INT NOT NULL, date_of_graduation DATE NOT NULL);
 CREATE TABLE specializations (vet_id INT REFERENCES vets (id), species_id INT REFERENCES species (id));
 CREATE TABLE visits (animal_id INT REFERENCES animals (id), vets_id INT REFERENCES vets (id), date_of_visit  DATE);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120); 
```

Comment:
- [ ]  it would be better to add indexes in the below tables and columns in order to reduce the time of the queries 
```SQL
 CREATE INDEX ON visits (animal_id);
 CREATE INDEX ON visits (vet_id);
 CREATE INDEX ON owners (email);
```